content/2018/bsdcan/lucas-ssh_key_management.md

```diff
@@ -3,6 +3,7 @@ layout: video
 title: "SSH Key Management"
```

Comment:
I like this one, but it might not a FreeBSD specified so I think it's better to check with the author first.

Reply:
I've sent an email to author.  In the meantime, I've removed it to let the pull request be approved.

Reply:
I think if we ask, we can get a blanket approval to post anything we find from Michael W. Lucas, but we should ask first.

Reply:
Michael Lucas emailed me "Sure, go ahead".  I'll add it back in the next pull request.

Reply:
my bad.  fixed.